### ISSUE TITLE
Period filter doesn't allow seconds,minutes as unit

### ISSUE THREAD

**Issue by hatharom** [elastic/curator#1549]
```
 filtertype: period 
            period_type: relative
            source: creation_date
            range_from: -120
            range_to: -1
            unit: minutes
```

`2020-07-18 22:24:49,341 ERROR     Failed to complete action: snapshot.  <class 'curator.exceptions.FailedExecution'>: Exception encountered.  Rerun with loglevel DEBUG and/or check Elasticsearch logs for more information. Exception: "unit" must be one of: ['hours', 'days', 'weeks', 'months', 'years']`

Although the documentation clearly states they are allowed : https://www.elastic.co/guide/en/elasticsearch/client/curator/current/filtertype_period.html#_relative_periods

**Comment by mattsdevop** [comment]
@hatharom the explanation is here: https://www.elastic.co/guide/en/elasticsearch/client/curator/current/fe_unit.html
"The values seconds and minutes are not allowed with the period filtertype and will result in an error condition if used there."

**Comment by untergeek** [comment]
Closed by #1550
